extensions/arc/package.nls.json

```diff
@@ -70,7 +70,7 @@
 
 
 	"resource.type.arc.sql.display.name": "Azure SQL managed instance - Azure Arc (preview)",
-	"resource.type.arc.postgres.display.name": "PostgreSQL server groups - Azure Arc (preview)",
+	"resource.type.arc.postgres.display.name": "PostgreSQL Hyperscale server groups - Azure Arc (Preview)",
```

Comment:
the casing of 'preview' should remain consistent across SQL and Postgress in my opinion.